hackathon_site/dashboard/frontend/src/slices/event/teamAdminSlice.ts

```diff
@@ -0,0 +1,98 @@
+import {
+    createAsyncThunk,
+    createEntityAdapter,
+    createSelector,
+    createSlice,
+} from "@reduxjs/toolkit";
+import { get } from "api/api";
+import { APIListResponse, Team } from "api/types";
+import { AppDispatch, RootState } from "slices/store";
+import { displaySnackbar } from "slices/ui/uiSlice";
+
+interface TeamAdminExtraState {
+    errorState: string | null;
+    isMoreLoading: boolean;
```

Comment:
you're not using this right now so I suggest you remove it. But if you want to keep it just make sure it's not interfereing with your current slice functionality and add a TODO saying it will be implemented in the future

Reply:
done